todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsPresenter.kt

```diff
@@ -45,9 +45,9 @@ class StatisticsPresenter(
         // that the app is busy until the response is handled.
         EspressoIdlingResource.increment() // App is busy until further notice
 
-        tasksRepository.getTasks(object : TasksDataSource.LoadTasksCallback {
-            override fun onTasksLoaded(tasks: List<Task>) {
-                // We calculate number of active and completed tasks
+        runBlocking {
```

Comment:
It is designed to bridge regular blocking code to libraries that are written in suspending style, to be used in main functions and in tests. You should use `launch`. See https://github.com/dmytrodanylyk/android-architecture/blob/dev-todo-mvp-kotlin-coroutines/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksPresenter.kt#L68

Also it's better to provide context explicitly.